package.json

```diff
@@ -69,6 +69,9 @@
     "qunit-dom": "^2.0.0",
     "webpack": "^5.74.0"
   },
+  "resolutions": {
+    "ansi-html": "^0.0.9"
```

Comment:
can this resolution be removed? I note that it was originally `^0.0.7` in the yarn.lock, so if we remove this again, yarn lock should remain at `^0.0.9`.
Can any of the other resolutions get the same treatment?